pkg/integration/tests/stash/drop.go

```diff
@@ -28,7 +28,7 @@ var Drop = NewIntegrationTest(NewIntegrationTestArgs{
 			Tap(func() {
 				t.ExpectPopup().Confirmation().
 					Title(Equals("Stash drop")).
-					Content(Contains("Are you sure you want to drop this stash entry?")).
+					Content(Contains("Are you sure you want to drop the selected stash entry(ies)?")).
```

Comment:
This change needs to be squashed into the previous commit, otherwise that commit doesn't have green tests.

But actually I would squash this entire commit into the previous one; in general I prefer tests to be added in the same commit that adds the feature.